front/app/components/PostCardsComponents/ViewButtons.tsx

```diff
@@ -28,7 +28,7 @@ const StyledIcon = styled(Icon)`
 `;
 
 const ViewButton = styled.button<{ active: boolean }>`
-  padding: 10px 12px;
+  padding: 4px 8px !important;
```

Comment:
We discussed this one. Let's check if !important is needed. Ideally, we shouldn't need it.